### PR TITLE
Feat: Onboard NHTSA Traffic Fatalities dataset

### PR DESCRIPTION
**Pull request by aurogoogle**
## Description

Dataset: NHTSA Traffic Fatalities

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

Use the sections below based on what's applicable to your PR and delete the rest:

### Feature
- [ ] I'm adding or editing a feature
- [ ] I have updated the [`README`](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/README.md) accordingly

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/<nhtsa_traffic_fatalities>` and nothing outside of that directory

### CODE REVIEW

**Comment by aurogoogle** [comment]
@adlersantos @nlarge-google Kindly review.

File: datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+            )
+            + ":"
+            + df["minute_of_crash"].apply(
+                lambda x: "0" + str(x) if len(str(x)) == 1 else str(x)
+            )
+            + ":"
+            + "00"
+            + " UTC"
+        )
+
```
Comment (nlarge-google) [inline datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py:338]: Please remove extra lines

File: datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+            target_file_batch, target_file, skip_header, not (skip_header)
+        )
+    elif new_pipeline_name in ["person", "vehicle"]:
+        create_new_timestamp_column(df, new_pipeline_name, process_year)
+    else:
+        save_to_new_file(df, file_path=str(target_file_batch), sep="|")
+        append_batch_file(
+            target_file_batch, target_file, skip_header, not (skip_header)
+        )
+
```
Comment (nlarge-google) [inline datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py:273]: Please remove extra lines

**Review by nlarge-google — CHANGES_REQUESTED** [review]
Great Job, just removal of blank lines.

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
```
[the remark this answers is not in the document]
Comment (aurogoogle) [reply Pipfile]: Removed the mentioned file from the commit / branch.

File: Pipfile.lock [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
[the remark this answers is not in the document]
Comment (aurogoogle) [reply Pipfile.lock]: Removed the mentioned file from the commit / branch.

File: datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 1 above)
Comment (aurogoogle) [reply datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py]: Removed extra spaces.

File: datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 2 above)
Comment (aurogoogle) [reply datasets/nhtsa_traffic_fatalities/pipelines/_images/run_csv_transform_kub/csv_transform.py]: Removed extra spaces.

**Comment by aurogoogle** [comment]
Hey @nlarge-google I have addressed the mentioned changes. Kindly review.

**Review by nlarge-google — APPROVED** [review]

**Comment by nlarge-google** [comment]
Great Job!

**Pull request merged**